Resources/Prototypes/Entities/Clothing/OuterClothing/hardsuits.yml

```diff
@@ -237,21 +237,21 @@
   - type: Clothing
     sprite: Clothing/OuterClothing/Hardsuits/rd.rsi
   - type: PressureProtection
-    highPressureMultiplier: 0.05
+    highPressureMultiplier: 0.02
     lowPressureMultiplier: 5500
   - type: Armor
     modifiers:
       coefficients:
         Blunt: 0.6
-        Slash: 0.75
+        Slash: 0.8
         Piercing: 0.95
-        Heat: 0.65
-        Radiation: 0.25
+        Heat: 0.3
+        Radiation: 0.1
   - type: ClothingSpeedModifier
     walkModifier: 0.75
     sprintModifier: 0.75
   - type: ExplosionResistance
-    damageCoefficient: 0.3
+    damageCoefficient: 0.1
```

Comment:
isnt the RD's suit supposed to be very bomb resistant. imo that makes sense as a niche for it more than protecting against rads (which the CE suit already does)

Reply:
oh wait im dumb lol

Reply:
> isnt the RD's suit supposed to be very bomb resistant. imo that makes sense as a niche for it more than protecting against rads (which the CE suit already does)

I did make it bomb resistant, did I not? It's 0.1 now, which is 90 percent DR. Radiation was because they'd be experimenting with artifacts and the like.

It's a steal objective - this means it's a high risk item. I assume that is because it is a good set of armour for the lab. I know radiation artifacts exist and therefore I gave it a high rad resistance. It's not radiation immune, like I made the CE suit.